vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -140,20 +146,27 @@ public Handler(ProjectDetailsComponent component,
 
     private void contactClickListener() {
       //when clicking the layout switch!
-      component.projectManagerLayout.addClickListener(click ->{
-        component.pmElement.setVisible(false);
+      component.projectManagerLayout.addClickListener(click -> {
         component.projectManagerComboBox.setVisible(true);
+        component.projectManagerComboBox.focus();
       });
 
-      //when selecting a pm switch back to the label element
-      component.projectManagerComboBox.addValueChangeListener(click -> {
-        PersonReference reference = click.getValue();
-        component.pmElement.setContent(reference.fullName(),reference.getEmailAddress());
-
+      component.projectManagerComboBox.addBlurListener(it -> {
+        PersonReference reference = projectManagerComboBox.getValue();
+        component.pmElement.setContent(reference.fullName(), reference.getEmailAddress());
```

Comment:
do we store the change in the project?

Reply:
yes this is triggered as for the other fields with another blur listener

Reply:
```java
      ProjectDetailsComponent.Handler.submitOnBlur(projectManagerComboBox,
          value -> projectInformationService.manageProject(selectedProject.value(), value));
```

Reply:
moved it to the value change listener